api/modules/user.js

```diff
@@ -0,0 +1,69 @@
+import db from '../db';
+import { v4 as uuidv4 } from 'uuid';
+
+const userTemplate = {
+    id: '',
+    login: '',
+    password: '',
+    age: null,
+    isDeleted: false
+};
+
+class User {
+    get(id) {
+        return db.users.find(({ id: userId }) => userId === id);
+    }
+
+    add(user) {
+        const newUser = {
+            ...userTemplate,
+            id: uuidv4(),
+            ...user
+        };
+        db.users.push(newUser);
+
+        return newUser;
+    }
+
+    update(id, user) {
+        const newUser = Object.assign({}, user);
+        delete newUser.id;
+        const index = db.users.findIndex(({ id: userId }) => userId === id);
```

Comment:
How about extracting this find operation in a separate method, as it is called twice?

Reply:
done